pkg/reconciler/common/ha.go

```diff
@@ -24,26 +24,9 @@ import (
 	v1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
 )
 
-func haSupport(obj v1alpha1.KComponent) sets.String {
+func unSupported(obj v1alpha1.KComponent) sets.String {
 	return sets.NewString(
-		"controller",
-		"autoscaler",
-		"autoscaler-hpa",
-		"networking-certmanager",
-		"networking-ns-cert",
-		"networking-istio",
-		"3scale-kourier-control",
-		"3scale-kourier-gateway",
-		"net-nscert-controller",
-		"net-certmanager-controller",
-		"net-istio-controller",
-		"net-kourier-controller",
-		"net-kourier-gateway",
-		"eventing-controller",
-		"sugar-controller",
-		"imc-controller",
-		"imc-dispatcher",
-		"mt-broker-controller",
+		"pingsource-mt-adapter",
```

Comment:
this would than also make the webhook to be HA, right?

those are currently only HPA:
* https://github.com/knative/operator/blob/main/cmd/operator/kodata/knative-serving/0.25.1/2-serving-core.yaml#L5434-L5447
* https://github.com/knative/operator/blob/main/cmd/operator/kodata/knative-eventing/0.25.1/2-eventing-core.yaml#L846-L864

Reply:
Yes, that's right.
webhook which has HPA is scaled up by the operator but other webhook are not be HA now. But all deployment including webhook will effect on the ha config.

Reply:
so we need to exclude the webhook than? Or we want it included, as it was missing before?

Reply:
I personally want to include the webhook. It is difficult to understand the current HA components that are effected by HA setting. So including all deployments and excluding unsupported components explicitly like `pingsource-mt-adapter` make it clear.

Reply:
I like this approach. We change it from a W-list to a B-list.

Reply:
`pingsource-mt-adapter` supports HA.